### PR TITLE
2003 Configure JDI BDD tests suites in Allure report

### PR DESCRIPTION
**Pull request by AkioFaris**
Added CucumberHooks to configure proper suites for BDD tests
- removed unnecessary for JUnit suiteXmlFile

### CODE REVIEW

File: jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package cucumberTests;
```
Comment (codacy-review) [inline jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java:1]: Codacy Issue found: [Package name contains upper case characters](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5557443)

**Comment by codacy-review** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- jdi-bdd-tests/src/test/java/cucumberTests/CucumberHooks.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5557443&bid=18227345)

**Comment by mf-code** [comment]
[5358] - V99yzRQN

**Comment by mf-code** [comment]
[5358] - ERjiDg8f

**Comment by mf-code** [comment]
[5358] - idHOcPTY

**Comment by mf-code** [comment]
[5358] - oQu1K4mv

**Comment by mf-code** [comment]
[5358] - XEiDt9YT

**Comment by mf-code** [comment]
[5358] - J5TATxkv

**Comment by mf-code** [comment]
[5358] - Allure report on Netlify: https://5ec5265993f3fef288122ec1--musing-swartz-6fc060.netlify.app

**Pull request merged**